specs/ProcessInfo.md

```diff
@@ -0,0 +1,251 @@
+Process Info
+===
+
+# Background
+Provide end developer a new API to get all browser and child process ID and type. End developers can use this API to do perf count, instead of trying to hook up with task manager API to do that. 
+
+# Examples
+## ProcessRequested
+
+Feature explanation text goes here, including why an app would use it, how it
+replaces or supplements existing functionality.
+
+```c#
+CoreWebView2ProcessCollection _processList;
+
+void WebView_CoreWebView2InitializationCompleted(object sender, CoreWebView2InitializationCompletedEventArgs e)
+{
+    if (e.IsSuccess)
+    {
+        // ...
+        WebViewEnvironment.ProcessInfoChanged += WebView_ProcessInfoChanged;
+    }
+}
+
+void WebView_ProcessInfoChanged(object sender, object e)
+{
+    _processList = WebViewEnvironment.ProcessInfo;
+}
+
+void ProcessInfoCmdExecuted(object target, ExecutedRoutedEventArgs e)
+{
+    string result;
+    uint processListSize = _processList.Count;
+    if (processListSize == 0)
+    {
+        result = "No process found.";
+    }
+    else
+    {
+        result = $"{processListSize} child process(s) found\n\n";
+        for (uint i = 0; i < processListSize; ++i)
+        {
+            uint processId = _processList.GetProcessIdAtIndex(i);
+            CoreWebView2ProcessKind kind = _processList.GetProcessTypeAtIndex(i);
+            result = result + $"Process ID: {processId}\nProcess Kind: {kind}\n";
+        }
+    }
+    MessageBox.Show(this, result, "Process List");
```

Comment:
We want to ensure sample code is actually doing something with our API that we imagine our end developers may want to do with it. Generally we want to avoid sample code that's just printing out the results of an API (unless that's what we want to do with this one?)

Reply:
Ideally it should be something end developers may want to do with the API. I don't think killing renderer processes is that. Are you able to use the perf counters APIs? 
* https://docs.microsoft.com/en-us/windows/win32/perfctrs/performance-counters-functions
* Or is there easier functions to use? https://docs.microsoft.com/en-us/windows/win32/api/psapi/nf-psapi-getprocessmemoryinfo

Reply:
I updated the sample code to also get memory usage using process id which is one kind of performance counter